customs.go

```diff
@@ -47,6 +50,70 @@ type createCustomsItemRequest struct {
 	CustomsItem *CustomsItem `json:"customs_item,omitempty"`
 }
 
+func (ci *CustomsItem) MarshalJSON() ([]byte, error) {
+	// convert a CustomsItem to JSON data byte array
+
+	// needed later to avoid infinite recursion
+	type Alias CustomsItem
+
+	return json.Marshal(&struct {
+		Value string `json:"value,omitempty"`
+		*Alias
+	}{
+		// always convert the Value into a string when serializing to JSON
+		Value: fmt.Sprintf("%f", ci.Value),
+		Alias: (*Alias)(ci),
+	})
+}
+
+func (ci *CustomsItem) UnmarshalJSON(b []byte) (err error) {
+	// convert a JSON data byte array to CustomsItem
+
+	// needed later to avoid infinite recursion
+	type Alias CustomsItem
+
+	// make a raw copy of the data
+	var rawData map[string]interface{}
+	if err = json.Unmarshal(b, &rawData); err != nil {
+		return
+	}
+
+	// tweak values
+	for k, v := range rawData {
+		switch k {
+		case "value":
+			// convert Value to a float64 if it is a string (via API response)
+			if s, ok := v.(string); ok {
+				if f, err := strconv.ParseFloat(s, 64); err == nil {
+					rawData[k] = f
+				}
+			} else {
+				// keep Value a float64 if it is already one (via Fixture data)
+				rawData[k] = v.(float64)
+			}
```

Comment:
```suggestion
	for key, value := range rawData {
		switch key {
		case "value":
			// convert Value to a float64 if it is a string (via API response)
			if stringValue, noErr := value.(string); noErr {
				if floatValue, err := strconv.ParseFloat(stringValue, 64); err == nil {
					rawData[key] = floatValue
				}
			} else {
				// keep Value a float64 if it is already one (via Fixture data)
				rawData[key] = value.(float64)
			}
```

Nit: improve the readability by expliciting the single character variable name.
Question: What if the `strconv.ParseFloat(s, 64)` failed or has error?

Reply:
Resolved.

If an error occurs internally, the method will return (an error)